docker/common/install_mooncake.sh

```
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -ex
 
+# Authenticate the github.com clones below; no-op when no token is available.
+source "$(dirname "${BASH_SOURCE[0]}")/github_auth.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./github_auth.sh was not specified as input (see shellcheck -x).

(SC1091)